.devcontainer/tpu-contributor/devcontainer.json

```diff
@@ -22,5 +22,6 @@
         "ms-python.python"
       ]
     }
-  }
+  },
+  "containerUser": "ptxla"
```

Comment:
This may break the builds or test executions, etc.. I think we may need sudo/root privilege for some of the operations. cc @will-cromar

Reply:
Will remove that.